setup.cfg

```diff
@@ -0,0 +1,7 @@
+[isort]
+known_first_party = cabot
+known_third_party = celery,jenkins,freezegun,twilio,requests
```

Comment:
Rather than sparse-specifying these 5 as third_party, let's rather set ``default_section` to `thirdparty`, maybe?